libs/components/ag-grid/src/lib/modules/ag-grid/cell-renderers/cell-renderer-template/cell-renderer-template.component.ts

```diff
@@ -0,0 +1,62 @@
+import { NgIf, NgTemplateOutlet } from '@angular/common';
+import { Component, TemplateRef } from '@angular/core';
+
+import { ICellRendererAngularComp } from 'ag-grid-angular';
+import { ICellRendererParams } from 'ag-grid-community';
+
+import { CellRendererTemplateContext } from './cell-renderer-template-context.type';
+
+type CellState =
+  | {
+      hasTemplate: true;
+      template: TemplateRef<CellRendererTemplateContext>;
+      context: CellRendererTemplateContext;
+    }
+  | {
+      hasTemplate: false;
+      template: undefined;
+      context?: unknown;
+    };
+
+@Component({
+  selector: 'sky-ag-grid-cell-renderer-template',
+  standalone: true,
+  template: `<ng-container *ngIf="state.hasTemplate">
+    <ng-container *ngTemplateOutlet="state.template; context: state.context" />
+  </ng-container>`,
+  imports: [NgTemplateOutlet, NgIf],
+})
+export class SkyAgGridCellRendererTemplateComponent
+  implements ICellRendererAngularComp
+{
+  protected state: CellState = {
+    hasTemplate: false,
+    template: undefined,
+    context: { value: undefined, data: undefined },
+  };
+
+  public agInit(params: ICellRendererParams): void {
+    const hasTemplate = !!(
+      params.colDef?.cellRendererParams &&
+      'template' in params.colDef.cellRendererParams &&
+      params.colDef.cellRendererParams.template
+    );
+    const template: TemplateRef<CellRendererTemplateContext> | undefined =
+      params.colDef?.cellRendererParams &&
```

Comment:
can we use the above `hasTemplate` here? looks like this condition is identical

Reply:
Good call, updated.